crypto/bls12381/bls12381.go

```diff
@@ -532,3 +411,7 @@ func (pubKey PubKey) Equals(other crypto.PubKey) bool {
 
 	return false
 }
+
+func errInvalidPrivateKeySize(size int) error {
```

Comment:
It's better to define custom type/var, so that we can easily compare errors later on.